src/abe_policy/policy.rs

```diff
@@ -586,7 +572,9 @@ mod tests {
         //
         // create policy
         let mut policy = policy()?;
+        println!("{policy:?}");
```

Comment:
Is these `println!()` useful during testing?

Reply:
Sorry, this slipped through my review...